_includes/field-description-payee-reference.md

```diff
@@ -1,6 +1,11 @@
 {%- capture documentation_section -%}{%- include documentation-section.md fallback="card" -%}{%- endcapture -%}
 {%- assign describe_receipt = include.describe_receipt | default: false -%}
 {%- capture documentation_section -%}{%- include documentation-section.md -%}{%- endcapture -%}
+{% if api_resource == "card" %}
```

Comment:
Where is `api_resource` defined? Should the `if` check here perhaps use `documentation_section` variable defined above (on line 1) instead?